aspnetcore/mvc/controllers/bind-tryparse/7.0-samples/BindUsingTryParse/BindTryParseMVC/Program.cs

```diff
@@ -15,6 +15,26 @@
     app.UseExceptionHandler("/Home/Error");
 }
 
+// GET /WeatherForecast/ByRange?range=7/24/2022,07/26/2022
+
+var redirectDateRange = $"/WeatherForecast/ByRange?range={DateTime.Now.ToShortDateString()}" +
+                        $",{DateTime.Now.AddDays(5).ToShortDateString()}";
+
+
+app.MapGet("/range", () => redirectDateRange);
+
+// GET /culture/en-GB
+// /en-GB/WeatherForecast/RangeByLocale&range=01/08/2022,06/08/2022
+app.MapGet("/culture/{cultureID}", (string cultureID) =>
+{
+    var cultureRange = $"/{cultureID}/WeatherForecast/RangeByLocale&range=" +
+                      $"{DateTime.Now.ToString("d",new CultureInfo(cultureID))}" +
+                      $",{DateTime.Now.AddDays(5).ToString("d",new CultureInfo(cultureID))}";
+    return cultureRange;
+});
```

Comment:
@fiyazbinhasan  I put this in to generate `RangeByLocale` test strings. But when I paste the output into the request I get a 404 error.
`https://localhost:50925/en-GB/WeatherForecast/RangeByLocale&range=01/08/2022,06/08/2022`

Reply:
Hi @Rick-Anderson, I assume you missed the controller template for the locale?

https://github.com/dotnet/AspNetCore.Docs/blob/fbca0eb370bea0592f27a9e5ba5cc2305f1d282e/aspnetcore/mvc/controllers/bind-tryparse/7.0-samples/BindUsingTryParse/BindTryParseMVC/Program.cs#L28-L30

Reply:
No, I pulled all your code down. Can you try that URL? Can you give me a URL to try?

Reply:
Got it. There is an `&` instead of `?` in the URL. The correct URL is,

`https://localhost:50925/en-GB/WeatherForecast/RangeByLocale?range=01/08/2022,06/08/2022`

Might I suggest you change the `MapGet` to this,

```
app.MapGet("locale/{localeId}", (string localeId) =>
{
    var cultureRange = $"/{localeId}/WeatherForecast/RangeByLocale?range=" +
                       $"{DateTime.Now.ToString("d", new CultureInfo(localeId))}" +
                       $",{DateTime.Now.AddDays(5).ToString("d", new CultureInfo(localeId))}";
    return cultureRange;
});
```

Or you can leave it like that 🐱